### ISSUE TITLE
some questions about the train of the ISIC2017

### ISSUE THREAD

**Issue by 1chijianren** [YonghengSun1997/MSCA-Net#10]
Hello author, I have a question regarding the code I downloaded from GitHub, which is designed for five-fold cross-validation and originally intended for the ISIC2018 dataset. If I want to train it on the ISIC2017 dataset instead, which doesn't require cross-validation, should I keep the folder_toy as it is in this part:
parser.add_argument('--val_folder', default='folder_toy', type=str, 
                    help='folder1, folder2, folder3, folder4, folder5')  # five-fold cross-validation
And only modify the following two paths to point to my ISIC2017 data:
parser.add_argument('--root_path', default='./data/ISIC2017_npy_all_224_320', 
                    help='root directory of data')  # The folder where the numpy dataset is stored

npy_path = os.path.join('./data/ISIC2017_npy_all_224_320/image', name[0])
Do I need to add additional code to load the test set? I noticed that the folder_toy text file only contains entries like ISIC_0000000.npy, and there doesn't seem to be any code specifying paths for the test or validation sets. Could you please advise on how to adapt this code for ISIC2017? Thank you very much for your time and help!
